src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -37,35 +37,43 @@ import { getConditionalPages } from '../../utilities';
 import SupportingDocsVerification from './supportingDocsVerification';
 import MissingFileList from './MissingFileList';
 
-const mailInfo = (
+const mailInfo = (showOpt = true, address, officeName, faxNum) => (
   <>
     <p>
       Your application will not be considered complete until VA receives all of
       your remaining required files.
     </p>
-    <p>
-      Optional files are not required to complete your application, but may
-      prevent delays in your processing time.
-    </p>
-    <br />
-    Mail your files to:
+    {showOpt ? (
+      <p>
+        Optional files are not required to complete your application, but may
+        prevent delays in your processing time.
+        <br />
+      </p>
+    ) : null}
+    Mail your documents to:
     <address className="vads-u-border-color--primary vads-u-border-left--4px vads-u-margin-left--3">
       <p className="vads-u-padding-x--10px vads-u-margin-left--1">
-        VHA Office of Community Care
-        <br />
-        CHAMPVA Eligibility
-        <br />
-        P.O. Box 469028
-        <br />
-        Denver, CO 80246-9028
-        <br />
-        United States of America
+        {address ?? (
+          <>
+            VHA Office of Community Care
+            <br />
+            CHAMPVA Eligibility
+            <br />
+            P.O. Box 469028
+            <br />
+            Denver, CO 80246-9028
+            <br />
+            United States of America
+          </>
+        )}
       </p>
     </address>
-    Or fax them to:{' '}
+    Or fax your documents here:
+    <br />
+    {officeName ?? ''},<br />
     <VaTelephone
       contact={JSON.stringify({
-        phoneNumber: '3033317809',
+        phoneNumber: faxNum ?? '3033317809',
         description: 'fax number',
       })}
     />
```

Comment:
Changed to a function so that any form using this shared component can supply the proper mailing/faxing information as needed.